pkg/flowaggregator/flowaggregator.go

```diff
@@ -431,7 +547,7 @@ func (fa *flowAggregator) flowRecordExpiryCheck(stopCh <-chan struct{}) {
 			if fa.externalFlowCollectorAddr != "" && fa.exportingProcess == nil {
 				err := fa.initExportingProcess()
 				if err != nil {
-					klog.Errorf("Error when initializing exporting process: %v, will retry in %s", err, fa.activeFlowRecordTimeout)
+					klog.ErrorS(err, "Error when initializing exporting process, will retry in %s", fa.activeFlowRecordTimeout)
```

Comment:
that's not really structured logging. Does it actually format correctly?

Reply:
No, it's incorrect. Fixed.